charts/wasmcloud-platform/README.md

```diff
@@ -0,0 +1,120 @@
+# wasmcloud-platform
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.2.0](https://img.shields.io/badge/AppVersion-1.2.0-informational?style=flat-square)
+
+[wasmCloud](https://wasmcloud.com/docs/intro) is an open source project from the Cloud Native Computing Foundation (CNCF) that enables teams to build polyglot applications composed of reusable Wasm components and run them—resiliently and efficiently—across any cloud, Kubernetes, datacenter, or edge.
+
+The wasmCloud Platform Helm Chart provides a turnkey solution, for running WebAssembly applications on Kubernetes.
+
+## Prerequisites
+
+*   Kubernetes: `>=1.24.0`
+
+*   Helm: `>=3.8.0`
+
+*   If installing the NATS server - a StatefulSet - the means of provisioing PVCs (e.g. a CSI driver)
+
+## Getting Started
+
+The wasmCloud platform is comprised of three components, a NATS cluster as the backbone of its [lattice](https://wasmcloud.com/docs/concepts/lattice), [Wadm](https://wasmcloud.com/docs/ecosystem/wadm) for WebAssembly applications lifecycle management, and wasmCloud [host](https://wasmcloud.com/docs/concepts/hosts), which can be provided either by wasmcloud-operator - if preferring managed hosts - or wasmcloud-host.
+
+To take full advantage of the chart features, it's best to use wasmcloud-operator, and install the chart in two steps.
+
+### Deploy wasmCloud Platform
+
+```bash
+helm upgrade --install \
+    --create-namespace \
+    wasmcloud-platform \
+    --values https://raw.githubusercontent.com/wasmCloud/wasmcloud/main/charts/wasmcloud-platform/values.yaml \
+    oci://ghcr.io/wasmcloud/charts/wasmcloud-platform \
+    --dependency-update \
+    --set "nats.enabled=true" \
+    --set "wadm.enabled=true" \
+    --set "operator.enabled=true" \
+    --set "host.enabled=false" \
+    --set "hostConfig.enabled=false"
```

Comment:
I think it might be good to set the stage by using a `values.yaml` file where these are set, and providing an example of said yaml file as part of the README, because managing individual flags gets tedious (and error prone) very quickly.

Reply:
This is how the default values are set too; the only reason that I use the set flag is to highlight which configuration attributes are involved in subchart inclusions and exclusions.
With that in mind, do you still prefer them removed?